lib/web_ui/lib/src/engine/html_image_codec.dart

```diff
@@ -39,10 +39,20 @@ class HtmlCodec implements ui.Codec {
       js_util.setProperty(imgElement, 'decoding', 'async');
       imgElement.decode().then((dynamic _) {
         chunkCallback?.call(100, 100);
+        int naturalWidth = imgElement.naturalWidth;
+        int naturalHeight = imgElement.naturalHeight;
+        // Workaround for https://bugzilla.mozilla.org/show_bug.cgi?id=700533.
+        if (naturalWidth == 0 && naturalHeight == 0 && (
+            browserEngine == BrowserEngine.firefox ||
+                browserEngine == BrowserEngine.ie11)) {
```

Comment:
Is there a reason we shouldn't apply the same logic for other browsers? Does this mean we'll let things crash if Chrome returns 0x0? If Chrome (and others) never return 0x0, then we don't need the browser check, do we?